packages/flutter_test/lib/src/matchers.dart

```diff
@@ -323,11 +323,11 @@ Matcher coversSameAreaAs(Path expectedPath, { @required Rect areaToCompare, int
 ///    verify that two different code paths create identical images.
 ///  * [flutter_test] for a discussion of test configurations, whereby callers
 ///    may swap out the backend for this matcher.
-AsyncMatcher matchesGoldenFile(dynamic key) {
+AsyncMatcher matchesGoldenFile(dynamic key, {String version}) {
```

Comment:
It feels kind of odd that the version is a String since we always pass an int. Why not make it int?

Reply:
That is an incredibly good point. 😜